contents/dotfiles/Dotfiles.md

```diff
@@ -0,0 +1,231 @@
+<frontmatter>
+  title: Introduction to Dotfiles
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
```

Comment:
can change pageNav to 4 so that the level 4 subheads `####` will also be in the page nav